ghost/webmentions/test/MentionSendingService.test.js

```diff
@@ -6,6 +6,7 @@ const externalRequest = require('../../core/core/server/lib/request-external.js'
 const sinon = require('sinon');
 const logging = require('@tryghost/logging');
 const {createModel} = require('./utils/index.js');
+const settingsCache = require('../../core/core/shared/settings-cache');
```

Comment:
For the tests, we would not need this and we generally add a fake settingsService to the service constructor. Here is an example of what I mean:
https://github.com/TryGhost/Ghost/blob/3ac282598c36eaed4196124393c360c6697ce2c9/ghost/email-service/test/email-renderer.test.js#L212-L218